### DISCUSSION TITLE
TextMeshPro Bug

### DISCUSSION THREAD

**Discussion by ttttttttttttttllllllllllllll** [Tualin14/baPlayer#30] [Development]
We don't know which words the user will use, so TextMeshPro causes the following.
![image](https://github.com/Tualin14/ArisStudio/assets/71857734/b69aec1b-5d0e-472a-9861-81c3316d026e)

**Comment by kiraio-moe** [comment]
The fonts that we used maybe didn't have some glyphs, so TMP used boxes as the default text to show.

**Reply by ttttttttttttttllllllllllllll** [reply]
> The fonts that we used maybe didn't have some glyphs, so TMP used boxes as the default text to show.

Is that the characters used in TMP are generated before construction, which is feasible for English with only 26 letters, but impossible for all Chinese characters.

We don't know which Chinese characters users will use.
